pkg/util/util.go

```diff
@@ -274,3 +276,19 @@ func GenerateKubeVirtGroupVersionKind(obj runtime.Object) (runtime.Object, error
 
 	return objCopy, nil
 }
+
+func GetVmiGuestMemory(spec v1.VirtualMachineInstanceSpec) (resource.Quantity, error) {
```

Comment:
This func is missing the case where `spec.Domain.Memory.Guest` is `nil` but `spec.Domain.Memory.Hugepages` is set?

Reply:
You're welcome!
TBH, I'm not sure I understand your question :)
Can you clarify?